README.md

```diff
@@ -76,3 +78,33 @@ You can find more detailed documentation on the [Slack hexdocs page].
 
 [RTM API page]: https://api.slack.com/rtm
 [Slack hexdocs page]: http://hexdocs.pm/slack/
+
+### External Usage
+
+If you want to do things like trigger the sending of messages outside of 
+your Slack handlers, you can leverage the `handle_info/3` callback to implement an
+external API:
+
+```elixir
+defmodule SlackRtm do
+  use Slack
+  def handle_info({:message, text, channel}, slack, state) do
+    IO.puts "Sending your message, captain!"
+    send_message(text, channel, slack)
+    {:ok, state}
+  end
+  def handle_info(_, _, state), do: {:ok, state}
+end
+```
+
+This allows you to not just respond to Slack RTM events, but programmatically
```

Comment:
Maybe also add, "Be sure your bot is in the general channel" or whatever wording makes the most sense.